kong/plugins/oauth2/access.lua

```diff
@@ -136,7 +135,9 @@ local function retrieve_scopes(parameters, conf)
     return false, {[ERROR] = "invalid_scope", error_description = "You must specify a " .. SCOPE}
```

Comment:
if the above `false` returns were changed to `nil`, perhaps this one should as well?